pkg/virt-handler/vm.go

```diff
@@ -1492,7 +1492,7 @@ func (d *VirtualMachineController) defaultExecute(key string,
 		if err != nil {
 			return err
 		}
-		if vmi.Status.Phase == phase {
+		if vmi.Status.Phase != phase {
```

Comment:
Thank you for the PR. However this condition is supposed to avoid hot-looping and is explicitly waiting for the phases to match. Did you encounter any issues around that?

Reply:
Sorry, I made a mistake